### ISSUE TITLE
TFException: Expects arg[0] to be string but float is provided 

### ISSUE THREAD

**Issue by nadir500** [Unity-Technologies/ml-agents#839]
hello 
i am using Speech Commands Model Exaple from TensorFlow with the variables: 
```
 string WAV_INPUT = "wav_data";
        string SOFTMAX_NAME = "labels_softmax";
        string[] outputScoresNames = new string[] { SOFTMAX_NAME };
```
and then add input to the model like this: 

```
  string[] labels = new string[] { "_silence_" , "_unknown_", "stop","go"};

         TextAsset model = Resources.Load("GoStop") as TextAsset;
         TFGraph  graph = new TFGraph();
         graph.Import(model.bytes);

        TFSession session = new TFSession(graph);

        var runner = session.GetRunner();

        runner.AddInput(graph[WAV_INPUT][0], audioFile);
        runner.Fetch(outputScoresNames);
        runner.Run();
```
and the error given for softmax is like that: 

```
TFException: Expects arg[0] to be string but float is provided
TensorFlow.TFStatus.CheckMaybeRaise (TensorFlow.TFStatus incomingStatus, System.Boolean last) (at <1fe2de69842a4a4ba15256b83cca05f3>:0)
TensorFlow.TFSession.Run (TensorFlow.TFOutput[] inputs, TensorFlow.TFTensor[] inputValues, TensorFlow.TFOutput[] outputs, TensorFlow.TFOperation[] targetOpers, TensorFlow.TFBuffer runMetadata, TensorFlow.TFBuffer runOptions, TensorFlow.TFStatus status) (at <1fe2de69842a4a4ba15256b83cca05f3>:0)
TensorFlow.TFSession+Runner.Run (TensorFlow.TFStatus status) (at <1fe2de69842a4a4ba15256b83cca05f3>:0)
tensor.recognize (System.Single[] audioFile) (at Assets/tensor.cs:51)
tensor.Start () (at Assets/tensor.cs:23)

```

**Comment by eshvk** [comment]
Hi @nadir500 I'd like to get some more context in terms of how this ties into ML-Agents. I am asking because this looks like a Tensorflow or Tensorflow Bindings issue and perhaps that community can find you an answer faster.

**Comment by nadir500** [comment]
i am using the model with the tensorflowsharp plugin in unity to make a call from it like passing an audio .wav file and figure out the keyword from the model so i was stuck in here.
i searched and asked for it...but no one encountered this error.

**Comment by dacostaleo** [comment]
Hi @nadir500 did you find a solution for this problem? i have the same. I need to use a string like an input for a pretrained model, but i cant find how to do that.

**Comment by Mr-DDDAlKilanny** [comment]
Maybe the problem is in your input / output names are not correct
